extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -17,6 +17,22 @@
 amp-carousel {
 }
 
+/**
+ * `amp-carousel-slide` class is added to every item in the carousel to
+ * allow page authors to target items for user-defined styling.
+ * Every `<amp-carousel>'s` immediate children is considered an item in
+ * the carousel.
+ */
+amp-carousel-slide {
+}
+
+.amp-carousel-slide .-amp-replaced-content {
```

Comment:
I am worried about cases where images are in wrapped in a top-level `div` or `figure` for various reasons. (like this page: https://cdn.ampproject.org/c/s/kansascity.relaymedia.com/amp/news/politics-government/article73988477.html)